src/main.cpp

```diff
@@ -3585,6 +3585,46 @@ void FallbackSproutValuePoolBalance(
     }
 }
 
+void FallbackTransparentValuePoolBalance(
```

Comment:
Almost a duplicated of `FallbackSproutValuePoolBalance` howver can maybe work on that in a separated PR.